container/test-impl-base/src/main/java/org/jboss/arquillian/container/test/impl/client/deployment/DeploymentGenerator.java

```diff
@@ -69,13 +69,19 @@
     private Instance<ProtocolRegistry> protocolRegistry;
 
     public void generateDeployment(@Observes GenerateDeployment event) {
-        DeploymentScenarioGenerator generator = serviceLoader.get().onlyOne(
-            DeploymentScenarioGenerator.class, AnnotationDeploymentScenarioGenerator.class);
+        //DeploymentScenarioGenerator generator = serviceLoader.get().onlyOne(
+        //    DeploymentScenarioGenerator.class, AnnotationDeploymentScenarioGenerator.class);
```

Comment:
So what do we do with this code? Resurrect or let it die?

Reply:
Die